pkg/operator/controller.go

```diff
@@ -205,26 +230,9 @@ func (c *Controller) sync() error {
 		cr.ResourceVersion = updatedCR.ResourceVersion
 	}
```

Comment:
As you extracted `syncOperatorStatus`, you can also extract `syncOperatorResource`.